From b5245a96bc282209a5b0a376d0b8248f664a6cd5 Mon Sep 17 00:00:00 2001
From: Nathan Froyd <froydnj@mozilla.com>
Date: Thu, 19 Dec 2019 21:41:23 +0000
Subject: [PATCH 19/35] Bug 1605215 - add install rules for `WASM_LIBRARY`;
 r=firefox-build-system-reviewers,rstewart

Differential Revision: https://phabricator.services.mozilla.com/D57871

--HG--
extra : moz-landing-system : lando

diff --git a/config/makefiles/target_binaries.mk b/config/makefiles/target_binaries.mk
index ec1081b0cedc..31b2c4367e94 100644
--- a/config/makefiles/target_binaries.mk
+++ b/config/makefiles/target_binaries.mk
@@ -24,6 +24,13 @@ endif
 INSTALL_TARGETS += SHARED_LIBRARY
 endif # SHARED_LIBRARY
 
+ifdef WASM_LIBRARY
+WASM_LIBRARY_FILES = $(WASM_LIBRARY)
+WASM_LIBRARY_DEST ?= $(FINAL_TARGET)
+WASM_LIBRARY_TARGET = target
+INSTALL_TARGETS += WASM_LIBRARY
+endif
+
 ifneq (,$(strip $(HOST_SIMPLE_PROGRAMS)))
 HOST_PROGRAMS_EXECUTABLES = $(HOST_SIMPLE_PROGRAMS) $(HOST_RUST_PROGRAMS)
 HOST_PROGRAMS_DEST ?= $(DIST)/host/bin
-- 
2.25.1

